de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/sweble/PlainTextConverter.java

```diff
@@ -161,6 +176,7 @@ protected WtNode before(WtNode node)
 		needSpace = false;
 		noWrap = false;
 		sections = new LinkedList<Integer>();
+		rows = new LinkedList<List<String>>();
```

Comment:
Usually, lists should by default be `ArrayList` and not `LinkedList`.

Reply:
Probably the explicit specification of the generic type is also not necessary.

Reply:
@reckart `LinkedList` was there before (see `sections`, thus I adapted this type here. Yet, I can change for this class - if you request it.

Reply:
If I encountered such a situation in foreign code, I might also leave it. If it was my own code I'd certainly change it. I'm happy to leave the decision to you how you prefer to handle it.

Reply:
in line _342_ we find `sections.removeLast()`. for this reason, sections must remain of type `LinkedList`. Nevertheless, I will change `rows` to `ArrayList` with the next commit.